.gitignore

```diff
@@ -1,2 +1,3 @@
 elm-stuff/
 node_modules/
+.idea
```

Comment:
You should add this to your global gitignore instead.

https://stackoverflow.com/questions/59328900/how-to-globally-ignore-the-idea-folder-in-git-not-just-adding-it-to-gitignor